manifests/rhoai/addon/apps/nvidia/gpu-quickstart.yaml

```diff
@@ -39,19 +39,19 @@ spec:
       description: |-
         Prerequisites:
 
-        - You have no notebook servers running in Red Hat OpenShift AI.
+        - You have no workbenches running in Red Hat OpenShift AI.
 
         1. Navigate to the Red Hat OpenShift AI dashboard.
-        2. Click **Launch** on the Jupyter card to open the **Start a notebook server** page.
+        2. Click **Launch application** on the Jupyter card to open the **Start a basic workbench** page.
```

Comment:
Same comment as line 28: Are we changing the Jupyter card to "Start basic workbench" card in ODH too? Based on @kaedward's doc we'll be changing it for RHOAI, but not sure if that is the same for ODH.